x-pack/plugins/security_solution/common/api/timeline/delete_note/delete_note_route.ts

```diff
@@ -10,4 +10,5 @@ import { unionWithNullType } from '../../../utility_types';
 
 export const deleteNoteSchema = runtimeTypes.partial({
   noteId: unionWithNullType(runtimeTypes.string),
+  noteIds: unionWithNullType(runtimeTypes.array(runtimeTypes.string)),
```

Comment:
If you can update the schema here as well please! https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/api/timeline/delete_note/delete_note_route_schema.yaml